src/display/canvas.js

```diff
@@ -1421,6 +1421,11 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       ctx.transform.apply(ctx, current.textMatrix);
       ctx.translate(current.x, current.y + current.textRise);
 
+      if (current.patternFill) {
+        // TODO: Some shading patterns are not applied correctly to text.
```

Comment:
Could you list the issue numbers here too for reference? Otherwise I'm afraid it will be hard to find those later.